### PR TITLE
WSL2 installation and configuration guide 

### PR DESCRIPTION
**Pull request by shreyasingh1**
Added `WSL2-install-instructions.md` in `brainlit/docs` and linked to this file in `README.md`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,6 +39,12 @@ The repository can now be considered a "holding bay" for code developed by Neuro
 
 ## Installation
 
+### Operating Systems
+Brainlit is compatible with iOS, Windows, and Unix systems.
```
Comment (tathey1) [inline README.md]: Mac, not iOS.

File: README.md [hunk 2]
```diff
@@ -39,6 +39,12 @@ The repository can now be considered a "holding bay" for code developed by Neuro
...
+
+#### Windows Linux Subsystem 2 
+For Windows 10 users that prefer Linux functionality without the speed sacrifice of a VM, Brainlit can be installed and run on WSL2. See installation walkthrough [here.](docs/WSL2-install-instructions.md)
```
Comment (tathey1) [inline README.md]: "virtual machine" instead of "VM"

**Review by tathey1 — CHANGES_REQUESTED** [review]
overall, great job

**Comment by shreyasingh1** [comment]
@tathey1 just updated, thanks!

**Review by bvarjavand — APPROVED** [review]
Great. Very thorough.

**Comment by bvarjavand** [comment]
This behavior implies that you need to pull the most recent changes from `develop` into your branch.

**Comment by shreyasingh1** [comment]
@bvarjavand hm it says I'm up to date with develop and the upstream develop, and branch changes should be pushed. Is there anything else I can try changing?

**Review by tathey1 — APPROVED** [review]

**Comment by bvarjavand** [comment]
see #12

**Pull request closed without merging**